packages/toolpad-app/src/toolpad/ToolpadShell.tsx

```diff
@@ -78,6 +86,81 @@ function UserFeedback() {
   );
 }
 
+function UpdateBanner() {
+  const [latestVersion, setLatestVersion] = React.useState(CURRENT_RELEASE_VERSION);
+  const [dismissedBanner, setDismissedBanner] = useLocalStorageState<boolean>(
+    `${CURRENT_RELEASE_VERSION}-update-banner-dismissed`,
```

Comment:
Hadn't thought about this problem yet, but this will keep pushing unused items in the local storage indefinitely